### DISCUSSION TITLE
Feature request: Stopping after active encoding process

### DISCUSSION THREAD

**Discussion by DerAffenkoenig** [RandomEngy/VidCoder#864] [Ideas]
If i encode a video with x265, my CPU-usage is only about 60%..... therefore i have to encode three files simultanously to get a 100% usage (that should be a "Handbrake"-Issue, not a "VidCoder"-Issue).

Now the Problem:
If i want to restart my system or whatever, i have to stop the encoding-process, but then i have to reencode the at this time active processes.

Feature request:
Adding an option to stop the encoding, after the active running tasks are done.
So, that the encoder finishes the three working processes and dont start a new one, after one of the three encodings are finished.

**Comment by RandomEngy** [comment]
VidCoder Beta has this feature:
![image](https://user-images.githubusercontent.com/9450861/126739813-8dc63ea1-0c58-46ec-9866-2b8df292c119.png)
